WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -14,3 +14,14 @@
     end
   end
 end
+
+describe "Momentjs activation" do
+  let(:file_content) { File.read(Rails.root.join('app', 'assets', 'javascripts', 'application.js')) }
+  let(:locales) { I18n.available_locales - [:en] }
+
+  (I18n.available_locales - [:en]).each do |locale|
+    context "for " + locale.to_s do
+      it { expect(file_content).to include("//= require moment/#{locale}.js") }
```

Comment:
Have you tried that it's informative enough anyone, when failing?
Maybe something like that would be more obvious
```ruby
it "moment/#{locale}.js is required in the application.js file" do
...
end
```